### PR TITLE
318 send in contribution button missing on competencedaysidemenu

### PR DESCRIPTION
**Pull request by techbecca**
Changes in CompetenceDaySideMenu
- Added a missing button
- Updated color on a button

Changes in general
- Remove font-weight adjustments in theme, to improve accessibility (contrast issues)

### CODE REVIEW

File: webapp/src/theme/Theme.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export const theme = createTheme({
       styleOverrides: {
         root: {
           textTransform: 'none',
-          fontWeight: 300,
+          // fontWeight: 300,
```
Comment (ipomega) [inline webapp/src/theme/Theme.ts:54]: These outcommented lines should probably be removed completely.

**Review by ipomega — APPROVED** [review]
LGTM!

**Review by Hannes-Samskog — APPROVED** [review]

File: webapp/src/theme/Theme.ts (hunk 1 above)
Comment (Hannes-Samskog) [reply webapp/src/theme/Theme.ts:54]: Yes I fix!

Comment (techbecca) [reply webapp/src/theme/Theme.ts]: Thanks, my bad!

**Pull request merged**